instrumentation/vaadin-14.2/javaagent/vaadin-14.2-javaagent.gradle

```diff
@@ -0,0 +1,39 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  fail {
+    group = "com.vaadin"
+    module = "flow-server"
+    versions = "[,2.2.0)"
+  }
+  pass {
+    group = "com.vaadin"
+    module = "flow-server"
+    versions = "[2.2.0,3)"
+  }
+  fail {
+    group = "com.vaadin"
+    module = "flow-server"
+    versions = "[3.0.0,3.1.0)"
+  }
+  pass {
+    group = "com.vaadin"
+    module = "flow-server"
+    versions = "[3.1.0,)"
+  }
+}
+
+dependencies {
+  compileOnly "com.vaadin:flow-server:2.2.0"
+
+  testLibrary 'com.vaadin:vaadin-spring-boot-starter:16.0.0'
+
+  testImplementation project(':instrumentation:vaadin-14.2:vaadin-testing')
+  testImplementation(project(':testing-common')) {
+    exclude(module: 'jetty-server')
+  }
+
+  testInstrumentation project(':instrumentation:servlet:servlet-3.0:javaagent')
+  testInstrumentation project(':instrumentation:servlet:servlet-common:javaagent')
```

Comment:
Please take a look that changes from https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2609 don't mess with you

Reply:
Rebased.